tests/models/jukebox/test_modeling_jukebox.py

```diff
@@ -335,10 +335,11 @@ def test_slow_sampling(self):
 
     @slow
     def test_fp16_slow_sampling(self):
-        model = JukeboxModel.from_pretrained(self.model_id, min_duration=0).eval().half().to("cuda")
-        labels = [i.cuda() for i in self.prepare_inputs(self.model_id)]
+        prior_id = "ArthurZ/jukebox_prior_0"
```

Comment:
This PR change the test from `JukeboxModel` to `JukeboxPrior`. I am not very familiar with Jukebox, probably this is good enough. But maybe if we can create a `JukeboxModel` with a single `JukeboxPrior` (that is from your `"ArthurZ/jukebox_prior_0"`), and test against `JukeboxModel`, it would be even better.

I leave you to make the final call @ArthurZucker

Reply:
BTW, is this "ArthurZ/jukebox_prior_0" no longer a real model but instead a randomly initialized model?

Reply:
Will leave it as is for simplicity 😉